agent/agent/process/tests/test_08_post_submission_checks.py

```diff
@@ -0,0 +1,166 @@
+# """Quality control checks that are performed after submission."""
```

Comment:
These look like they could be useful. Perhaps they've been incorporated in other tests already. If not, could they be reenabled?

Reply:
Actually those were rewritten into the other tests in ``agent/agent/process/tests/`` -- removed this file in a059a01